atrium/AtriumClient.php

```diff
@@ -215,6 +217,15 @@ function listMemberTransactions($userGUID, $memberGUID) {
     return $transactionArray;
   }
 
+  function verifyMember($userGUID, $memberGUID) {
+    $response = $this->makeRequest('POST', '/users/' . $userGUID . '/members/' . $memberGUID . '/verify', '');
```

Comment:
I am pretty sure you can do string formatting like this:

```php
$response = $this->makeRequest('POST', "/users/$userGUID/members/$memberGUID/verify", '');
```
I am also pretty sure that double quotes are required for that.

Reply:
This is correct.